cropharvest/config.py

```diff
@@ -22,6 +25,14 @@
 FEATURES_DIR = "features"
 TEST_FEATURES_DIR = "test_features"
 
+# These values describe the structure of the data folder
```

Comment:
@ivanzvonkov , this locks in the folder structure but I think that's fine.

We could potentially have a way of over-riding this datafolder path at a package level, but otherwise I'd be for removing folder manipulation for the user entirely and controlling it here.